From 86431b83e39fb3c0eb1ebdc24173988f0da1192d Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Fri, 25 Jan 2013 15:16:07 +0800
Subject: [PATCH 11/70] ENGR00241755 FSLOTA: fix a error cause not read
 config.

fix a typo that not read the /etc/ota.conf.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 FSLOta/src/com/fsl/android/ota/OTAServerConfig.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/FSLOta/src/com/fsl/android/ota/OTAServerConfig.java b/FSLOta/src/com/fsl/android/ota/OTAServerConfig.java
index e2a5eed..e579942 100644
--- a/FSLOta/src/com/fsl/android/ota/OTAServerConfig.java
+++ b/FSLOta/src/com/fsl/android/ota/OTAServerConfig.java
@@ -58,7 +58,7 @@ public class OTAServerConfig {
 			return false;
 		}
 		
-		return false;
+		return true;
 	}
 	
 	void defaultConfigure(String productname) throws MalformedURLException
-- 
1.8.0

